plugins/core/hu.bme.mit.gamma.statechart.language/src/hu/bme/mit/gamma/statechart/language/scoping/StatechartLanguageScopeProvider.java

```diff
@@ -193,6 +194,19 @@ public IScope getScope(final EObject context, final EReference reference) {
 				List<State> states = StatechartModelDerivedFeatures.getStates(region);
 				return Scopes.scopeFor(states);
 			}
+			if (context instanceof InterfaceParameterReferenceExpression
```

Comment:
Remove duplication by introducing a single if which contains the other ifs (note the cast in the condition):
`
if (context instanceof InterfaceParameterReferenceExpression interfaceParameterReferenceExpression) {
`